test/sqllogictest/cockroach/aggregate.slt

```diff
@@ -871,29 +871,34 @@ true
 statement ok
 CREATE TABLE bools (b BOOL)
 
-query error supported
+query BB
 SELECT bool_and(b), bool_or(b) FROM bools
+----
+NULL NULL
 
 statement OK
 INSERT INTO bools VALUES (true), (true), (true)
 
-query error supported
+query BB
 SELECT bool_and(b), bool_or(b) FROM bools
+----
+true true
 
 statement OK
 INSERT INTO bools VALUES (false), (false)
 
-query error supported
+query BB
 SELECT bool_and(b), bool_or(b) FROM bools
+----
+false true
 
-# not supported yet
-# statement OK
-# DELETE FROM bools WHERE b
-#
-# query BB
-# SELECT bool_and(b), bool_or(b) FROM bools
-# ----
-# false false
+statement OK
+DELETE FROM bools WHERE b
+
+query BB
+SELECT bool_and(b), bool_or(b) FROM bools
+----
+false false
```

Comment:
I think we want at a few additional tests:

- `true, false, NULL`
- `true, true, NULL`
- `false, false, NULL`

I think the subtle one is the second, for which `bool_and` returns `true`